tools/build-agent-release.sh

```diff
@@ -0,0 +1,54 @@
+RELEASEDIR=releases
+TAG=$(date +%Y%m%d)-0.$(git log --pretty=format:'%h' -n 1)
+
+BUILDS=(
+  # FORMAT
+  # GOOS GOARCH BINSUFFIX
+  "darwin 386 darwin-i386-$TAG.prod"
```

Comment:
What meaning is intended by “prod”? Typically I see it paired with “dev” and/or “stage”, in which case this would benefit from being moved to a variable at the top of the script so that it’s not a buried string constant.